test/tools/libtesteth/TestOutputHelper.h

```diff
@@ -62,6 +66,7 @@ class TestOutputHelper
 	boost::filesystem::path m_currentTestFileName;
 	typedef std::pair<double, std::string> execTimeName;
 	std::vector<execTimeName> m_execTimeResults;
+    std::map<std::string, bool> m_testSuiteStatus;
```

Comment:
What's the "status"? I know now after reading the code, but it's not clear at all when you just look at this.

Alternative design suggestion: make this `std::vector<std::string> m_finishedTestSuites`; get rid of `registerTestFolders`; instead collect this list of test suit folders in the end when it's finished, put it into another `std::vector<std::string>`, use `std::set_difference` to find the folders present on disk but absent in `m_finishedTestSuites`

Reply:
thanks!